src/static/images/intro-background-fit.svg

```diff
@@ -1,13 +1,12 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="1920" height="1250" viewBox="0 0 1920 1250" preserveAspectRatio="none">
+<svg xmlns="http://www.w3.org/2000/svg" width="1920" height="1250" viewBox="0 0 1920 1250" preserveAspectRatio="none" style="background:white;">
```

Comment:
For consistency,. maybe move the inline style within the inner SVG's style .e.g. `svg {background: #fff}`?

Reply:
Cool. Will have a look at this. Not overly familair with SVGs so great feedback!

Reply:
Fixed!